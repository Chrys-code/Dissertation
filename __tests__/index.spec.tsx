import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import Home from '../pages/index';


test('true is truthy', () => {
  expect(true).toBe(true);
});

test('false is falsy', () => {
  expect(false).toBe(false);
});

describe('Index', () => {
  it('should render without throwing an error', () => {
    const testRenderer = TestRenderer.create(<Home />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});