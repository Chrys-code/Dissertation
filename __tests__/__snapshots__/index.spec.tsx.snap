// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render without throwing an error 1`] = `
Array [
  <div
    className="navbar"
  >
    <div
      className="container"
    >
      <button
        className="btn"
        type="submit"
      >
        ‚Üê
      </button>
      <div
        className="profile"
      >
        <p
          id="profile_name"
        >
          Name here
        </p>
        <button
          className="btn"
          type="submit"
        >
          <img
            alt="avatar"
            src="/images/avatar.svg"
          />
        </button>
      </div>
    </div>
    <div
      className="prespad"
    >
      <h1>
        Profile
      </h1>
      <h2>
        Name:
      </h2>
      <p>
        Sample
      </p>
      <h2>
        With children:
      </h2>
      <p>
        Sample
      </p>
      <h2>
        Departure location:
      </h2>
      <p>
        Sample
      </p>
      <h2>
        Departure time:
      </h2>
      <p>
        Sample
      </p>
      <h2>
        Arrival location:
      </h2>
      <p>
        Sample
      </p>
      <h2>
        Arrival time:
      </h2>
      <p>
        Sample
      </p>
      <h2>
        Transport name, number:
      </h2>
      <p>
        Sample
      </p>
      <div
        className="prespad"
      >
        <img
          alt="qr_code_smaple"
          src="/images/qr.svg"
        />
        
        
      </div>
      
      
      
    </div>
  </div>,
  <div
    className="container"
  >
    <div
      className="diagram"
    >
      <div
        className="prespad"
      >
        <select
          className="dropdown"
        >
          <option
            value="Country"
          >
            Country
          </option>
          <option
            value="123"
          >
            123
          </option>
        </select>
        <p>
          We should have an integrated Python plotlib diagram here
        </p>
        
        
        
      </div>
    </div>
    <a
      href="/form"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="prespad"
      >
        <img
          alt="green_backgroun_img"
          src="/images/locator.svg"
        />
        <h2>
          Locator From
        </h2>
        <p>
          Fill a form that is required to cross country border
        </p>
      </div>
    </a>
    <a
      href="/links"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="prespad"
      >
        <img
          alt="blue_backgroun_img"
          src="/images/links.svg"
        />
        <h2>
          Informations
        </h2>
        <p>
          See a list of useful links, or collect yourself
        </p>
      </div>
    </a>
  </div>,
]
`;
